src/protocol.cpp

```diff
@@ -241,3 +241,126 @@ std::vector<std::string> serviceFlagsToStr(uint64_t flags)
 
     return str_flags;
 }
+
+uint8_t GetShortCommandIDFromCommand(const std::string& cmd)
```

Comment:
I would prefer to return an `Option<uint8_t>` here to make it clearer when there is no short id available for a command (instead of special-casing `0`).

Reply:
> A single comment specifying this is used for V2 messaging protocol as per BIP324 might help here.

The comment is in the `.h` file, where it belongs for public functions.